msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -409,6 +419,71 @@ The default value of the property `isUserPersonalScopeResourceSpecificConsentEna
 
 <br>
 
+<b> After March 14th, 2024 </b>
+
+Admins will adjust Team RSC and Chat RSC settings via PowerShell cmndlets. Below are the available states for the PowerShell settings. Each section below will show examples of how to use these states to adjust your settings.
+
+| PowerShell State | Description |
+| ---- | :----: |
+| ManagedByMicrosoft |  This is the default state for all tenants, it allows Chat RSC and Team RSC permissions to be consented for all users, but can be changed at any time at Microsofts discretion |
+| ApprovedForAllApps |  Any RSC enabled app can be consented to by users in your tenant. With these settings enabled, preapproval of RSC permissions is not needed as all RSC enabled apps are approved for use |
```

Comment:
Any app requesting RSC permissions can be consented to by users (resource owners) in your tenant.